src/application/components/Enum.js

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
Wonder if we could just adapt Model.js to support enums? Looks like Model only has one extra div?